automation/project/Dependencies.scala

```diff
@@ -41,10 +41,9 @@ object Dependencies {
     "com.typesafe.akka" %% "akka-http" % akkaHttpV,
     "com.typesafe.akka" %% "akka-testkit" % akkaV % "test",
     "com.typesafe.akka" %% "akka-slf4j" % akkaV,
-    "org.scalatest" %% "scalatest" % "3.2.3" % Test,
     "com.typesafe.scala-logging" %% "scala-logging" % "3.9.2",
-    "org.scalatest" %% "scalatest" % "3.2.12" % Test,
-    "org.scalatestplus" %% "scalacheck-1-15" % "3.2.11.0" % Test,
+    "org.scalatest" %% "scalatest" % "3.2.19" % Test,
+    "org.scalatestplus" %% "scalacheck-1-18" % "3.2.19.0" % Test,
```

Comment:
`"org.scalatest" %% "scalatest"` was listed twice - with two different versions! I removed one of them.